src/index.js

```diff
@@ -115,10 +115,26 @@ const main = async () => {
     post = await db.collection('posts').findOne({ _id: id })
     res.status(201).send(post)
   })
-  // retreive news and comment sorted by timestamp in reversed order
+  // retreive news and comment sorted by wilson score or timestamp in reversed order
   app.get('/posts', async (req, res) => {
-    const timeline = await db.collection('posts').find().sort({ timestamp: -1 }).toArray()
-    res.json(timeline)
+    let list
+    switch (req.query.params) {
```

Comment:
You should give this query parameter a meaningful name. Maybe `req.query.sortBy`.